### PR TITLE
Admin invoice

### PR DESCRIPTION
**Pull request by Malllll12**
Almost done with the Admin Invoice  show page. After this I'm going to pivot and start on the Admin Dashboard AR story.

### CODE REVIEW

**Comment by Malllll12** [comment]
Gotta work on making more commits. I'm mostly only doing them once the story is done. Anybody have a different workflow?

File: app/views/admin/invoices/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,7 @@
-<h1>Invoice ID</h1>
+<h1>Invoice ID's</h1>
```
Comment (hannahkwarren) [inline app/views/admin/invoices/index.html.erb:1]: Grammar thing (feel free to change later, it doesn't need to hold up this PR): 
If the header on this page is just referring to "all the ids" - you don't need the apostrophe, should be "Invoice IDs"

**Comment by hannahkwarren** [comment]
I've been trying to remember to commit when I've created the tests I need, and then when those tests are passing in the one file. Though I probably only remember 50% of the time... wouldn't worry too much about it.

File: app/views/admin/invoices/index.html.erb (hunk 1 above)
Comment (Malllll12) [reply app/views/admin/invoices/index.html.erb:1]: Good point! Thanks for catching that.

**Pull request merged**